homeassistant/components/sensor/nest.py

```diff
@@ -16,7 +16,7 @@
 
 DEPENDENCIES = ['nest']
 SENSOR_TYPES = ['humidity',
-                'operation_mode',
+                'mode',
```

Comment:
No, we changed this on purpose. See breaking changes https://home-assistant.io/blog/2016/09/29/async-sleepiq-emoncms-stocks/#breaking-changes